lib/output.js

```diff
@@ -69,11 +69,15 @@ function OutputStream(options) {
         preamble         : null,
         quote_style      : 0,
         keep_quoted_props: false,
+        shorthand        : undefined,
         ecma             : 5,
     }, true);
     if (typeof options.ascii_identifiers === 'undefined')
         options.ascii_identifiers = options.ascii_only;
```

Comment:
I recognize this is unrelated to your patch, but could you also put a blank line above the `if` on line 75 as well?